docs/FAQ.md

```diff
@@ -0,0 +1,23 @@
+# FAQ
+
+**What does OA stand for?**
+
+ocm-agent, also OAO stands for ocm-agent-operator.
+
+**What is ocm-agent?**
+
+ocm-agent is a web-service managed by OAO as a funnel point for all managed clusters related to any OCM interaction. 
+
+**What are ocm-agent use-cases?**
+
+1. Provide a service endpoint for the publishing of service logs to the customer from other on-cluster workloads.
+
+> **_NOTE_** OCM Agent use-cases will grow as the project adds abilietis.
+
+**Does OA silence alerts?**
+
+No. Alert on-boarded to OCM Agent will not reach Pager-Duty but will have a service-log sent automatically.
```

Comment:
I think we will also silence the alert? Not in Pagerduty, but we "silence" them in alertmanager with redirect the alerts to other receiver.

Reply:
Ack, instead of mentioning "OA doesn't silence" I can put "We silence alerts but in Alertmanager instead of PD". WDYT?

Reply:
```Instead of sending the alerts to PagerDuty and page the primary, the OCM Agent will redirect the selected alerts to the embedded alertmanager-receiver, and have the service-log sent automatically.```

Something like above, you can polish the description based on that.

Reply:
Done.